functions/go/apply-replacements/README.md

```diff
@@ -0,0 +1,107 @@
+# apply-replacements
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Invoke the [kustomize replacements] feature as a KRM function. 
+
+<!--mdtogo-->
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+We use a `Replacements` object to configure the `apply-replacements` function. 
+
+```yaml
+apiVersion: kpt.dev/v1alpha1
+kind: Replacements
+metadata:
+  name: replacements-fn-config
+replacements:
+  # your replacements here
+```
+
+The syntax for `replacements` is described in the [kustomize replacements] docs. 
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+Suppose you have the following Job:
```

Comment:
It seems this example is identical to the one in examples dir.
If that's the case, we can emit the examples section here.

Reply:
done